crates/javy/src/runtime.rs

```diff
@@ -31,15 +37,96 @@ pub struct Runtime {
 
 impl Runtime {
     /// Creates a new [Runtime].
-    pub fn new(_config: Config) -> Result<Self> {
+    pub fn new(config: Config) -> Result<Self> {
         let rt = ManuallyDrop::new(QRuntime::new()?);
 
         // See comment above about configuring GC behaviour.
         rt.set_gc_threshold(usize::MAX);
-        let context = ManuallyDrop::new(Context::full(&rt)?);
+        let context = Self::build_from_config(&rt, &config)?;
         Ok(Self { inner: rt, context })
     }
 
+    fn build_from_config(rt: &QRuntime, cfg: &Config) -> Result<ManuallyDrop<Context>> {
+        let intrinsics = &cfg.intrinsics;
+        let javy_intrinsics = &cfg.javy_intrinsics;
+        // We always set Random give that the principles around snapshotting and
+        // random are applicable to all uses of Javy, given that the use of
+        // Wizer is not optional.
```

Comment:
Is Wizer not being optional true in the case of someone using this crate as a library without using our CLI?

That said it's not a big deal since the outcome of calling `Math.random()` should be identical in the scenario where they choose not to Wizen.

Reply:
Good point! I've amended the wording in the comment.